Tests/PerceptionTests/PerceptionTests.swift

```diff
@@ -0,0 +1,93 @@
+import Perception
+import SwiftUI
+import XCTest
+
+@available(iOS 16, macOS 13, tvOS 16, watchOS 9, *)
+@MainActor
+final class PerceptionTests: XCTestCase {
+  func testRuntimeWarning_NotInPerceptionBody() {
+    let model = Model()
+    model.count += 1
+    XCTAssertEqual(model.count, 1)
+  }
+
+  func testRuntimeWarning_InPerceptionBody_NotInSwiftUIBody() {
+    let model = Model()
+    PerceptionLocals.$isInPerceptionTracking.withValue(true) {
+      _ = model.count
+    }
+  }
+
+  func testRuntimeWarning_NotInPerceptionBody_InSwiftUIBody() async {
+    if #unavailable(iOS 17) {
+      XCTExpectFailure {
+        $0.compactDescription == """
+          Perceptible state was accessed but is not being tracked. Track changes to state by \
+          wrapping your view in a 'WithPerceptionTracking' view.
+          """
+      }
+    }
+    struct FeatureView: View {
+      let model = Model()
+      var body: some View {
+        Text(self.model.count.description)
+      }
+    }
+    _ = ImageRenderer(content: FeatureView()).uiImage
```

Comment:
Rendering a SwiftUI view to an image is a quick way to force all the attribute graph stuff to execute in a unit test so that we don't have to resort to an integration test